spec/classes/init_spec.rb

```diff
@@ -80,7 +80,7 @@
         'enable'      => true,
         'hasrestart'  => true,
         'hasstatus'   => true,
-        'require'     => 'Class[Apache::Mod::Shib]'
+        'require'     => '[Class[Apache::Mod::Shib],User[_shibd]'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.